frontend/src/components/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -48,7 +48,9 @@ const SignInForm: FC<Props> = ({ onSubmit }) => {
           />
         </div>
         <div className={styles.buttonWrapper}>
-          <Button type="submit" label="Sign In" btnColor="blue" />
+          <div>
+            <Button type="submit" label="Sign In" btnColor="blue" />
+          </div>
```

Comment:
Why do we need this wrapper?
```suggestion
          <Button type="submit" label="Sign In" btnColor="blue" />
```

Reply:
It's the button wrapper, the parent of which has display: flex. I wanted to add align-self to it, but there was no need, since flex is doing it all. The reason I was adding display: flex is that align-self would have no effect, without the flex container.